recipe/meta.yaml

```diff
@@ -53,13 +52,14 @@ test:
         - celery.utils
         - celery.utils.dispatch
         - celery.worker
-        
+
     commands:
         - celery --version
         - celery worker --help
         - celery beat --help
         - celery multi help
         - celery shell --help
+        - pytest t/unit  # [py3k]
```

Comment:
Covering tests was I thought usually not the conda-forge style. Maybe drop this? (cc @kwilcox)